homeassistant/components/unifi/controller.py

```diff
@@ -59,9 +68,38 @@ def site_role(self):
         return self._site_role
 
     @property
-    def block_clients(self):
-        """Return list of clients to block."""
-        return self.unifi_config.get(CONF_BLOCK_CLIENT, [])
+    def option_block_clients(self):
+        """Config entry option with list of clients to control network access."""
+        return self.config_entry.options.get(CONF_BLOCK_CLIENT, [])
+
+    @property
+    def option_dont_track_clients(self):
+        """Config entry option to not track clients."""
+        return self.config_entry.options.get(CONF_DONT_TRACK_CLIENTS, False)
```

Comment:
please extract these defaults into constants so you can share them between this code and the options flow.